tilec/fg.py

```diff
@@ -1,11 +1,84 @@
+from __future__ import print_function
 import numpy as np
+import yaml
 
-# Set up SZ frequency dependence
-def gnu(nu_ghz,tcmb=2.7255):
-    nu = 1e9*np.asarray(nu_ghz)
-    hplanck = 6.62607e-34
-    kboltzmann = 1.38065e-23 
-    x = hplanck*nu/kboltzmann/tcmb
-    coth = np.cosh(x/2.)/np.sinh(x/2.)
-    return x*coth-4.
+######################################
+# global constants
```

Comment:
We should probably move these to a yaml as well.

Reply:
sure, feel free.  I assume there are more global constants in use than just these